k8s.gcr.io/k8s-staging-e2e-test-images/manifest.yaml

```diff
@@ -0,0 +1,12 @@
+# google group for gcr.io/kubernetes-staging-e2e-test-images is k8s-infra-staging-e2e-test-images@kubernetes.io
+registries:
+- name: gcr.io/kubernetes-staging-e2e-test-images
```

Comment:
This would need to be `k8s-staging-e2e-test-images` to be consistent with other names, and also because the maximum number of characters in the name can only be 30.

Reply:
Done